tests/structure/test_charges.py

```diff
@@ -356,4 +378,98 @@ def test_correct_output_ions():
     sodium_array.bonds = BondList(sodium_array.array_length())
     with pytest.warns(None) as record:
         partial_charges(sodium_array, iteration_step_num=1)
-    assert len(record) == 0
\ No newline at end of file
+    assert len(record) == 0
```

Comment:
Please also check for the correctly calculated partial charge

Reply:
Done.